src/app/sb-admin/components/pages/sb-user/sbuser.component.html

```diff
@@ -63,12 +63,35 @@ <h5 class="m-0">{{ 'COMMON_USER' | i18next}}</h5><br>
                         <td>
                             <div class="flex">
                                 <button [disabled]="!user.status" pButton pRipple icon="pi pi-pencil" class="p-button-rounded p-button-info mr-2" (click)="editRole(user)"></button>
-                                <button pButton pRipple icon="pi pi-trash" class="p-button-rounded p-button-warning"></button>
+                                <button pButton pRipple icon="pi pi-trash" class="p-button-rounded p-button-warning" (click)="blockUser(user)" *ngIf="user.status"></button>
+                                <button pButton pRipple icon="pi pi-unlock" class="p-button-rounded p-button-warning" (click)="unblockUser(user)" *ngIf="!user.status"></button>
                             </div>
                         </td>
                     </tr>
                 </ng-template>
             </p-table>
+
+            <p-dialog [(visible)]="deleteUserDialog" header="Confirm" [modal]="true" >
```

Comment:
@snehalsabade can we use only one p-dialog for block and unblock user

Reply:
resolved